.github/workflows/test-go.yml

```diff
@@ -476,6 +476,7 @@ jobs:
           name: test-results
           path: test-results/go-test
       - run: |
+          rm -rf test-results/go-test/logs
```

Comment:
I think putting this line here would prevent us from having any logs in our test results.  Why not put it instead in sanitize-timing-files, i.e. right after the cache restore?

Reply:
I'm reconsidering my alternate suggestion, it doesn't sound quite right either.  Plus, test-collect-reports runs after "Archive test results", so my concern is probably bogus.

Reply:
Yeah it's fine, we have the logs in the archive.